CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 0.6.37 (July 30, 2024)
+* BUGFIX: [#446](https://github.com/opsgenie/terraform-provider-opsgenie/pull/446)
+  * **Integration Policy**
+    * Fixed perpetual time drift in the conditions field in Integration policy
+## 0.6.36 (July 7, 2024)
```

Comment:
Please add everything mentioned in `What's Changed` from https://github.com/opsgenie/terraform-provider-opsgenie/releases